docs/ldml/tr35.md

```diff
@@ -4077,70 +4077,44 @@ Other contributors to CLDR are listed on the [CLDR Project Page](https://www.uni
 **Differences from LDML Version 44.1**
 
 * Part 1: [Core](tr35.md#Contents)
-  * In [Parent Locales](tr35.md#Parent_Locales), add an additional attribute and value `localeRules="nonlikelyScript"` to allow for algorithmic handling of inheritance so that mixtures of scripts are avoided.
-  For example, preventing `ru_Latn` from falling back to `ru` (which is Cyrillic script). 
-  Previously, there was a small list of such locales, but they were far from covering all cases. 
-  That list was retained for migration.
+  * In [Parent Locales](#Parent_Locales), added an additional attribute and value `localeRules="nonlikelyScript"` 
+    to allow for algorithmic handling of inheritance so that mixtures of scripts are avoided.
+    For example, preventing `ru_Latn` from falling back to `ru` (which is Cyrillic script). 
+    Previously, there was a small list of such locales, but they were far from covering all cases. 
+    That list was retained for migration.
+  * In [Special Script Codes](#special-script-codes), added a description of special script codes,
+    such as Jpan and Aran.
+  * In [Lateral Inheritance](#Lateral_Inheritance), improved the formatting for clarity.
+  * In [Parent_Locales](#Parent_Locales), substantial changes to the way that parentLocales work,
```

Comment:
This bullet seems to mostly duplicate the updated first bullet above, perhaps they should be merged.

Reply:
Merged them, thanks!